DevelopGitHub/GitHubExtension/GitHubExtension.Infrastructure.ActionFilters/Extensions/ActivityFiltersExtensions.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Web.Http.Dependencies;
+using System.Web.Http.Filters;
+using GitHubExtension.Infrastructure.ActionFilters.Constants;
+using GitHubExtension.Infrastructure.ActionFilters.Models;
+using Microsoft.AspNet.Identity;
+
+namespace GitHubExtension.Infrastructure.ActionFilters.Extensions
+{
+    public static class ActivityFiltersExtensions
+    {
+        public static IDependencyResolver GetDependencyResolver(this HttpActionExecutedContext actionExecutedContext)
+        {
+            return actionExecutedContext.ActionContext.RequestContext.Configuration.DependencyResolver;
+        }
+
+        public static string GetUserId(this HttpActionExecutedContext actionExecutedContext)
```

Comment:
You are using this method only in one place. Why it should be public? Incapsulate all unnessesary information that is not available for client.